instrumentation/apache-httpasyncclient-4.1/javaagent/src/test/groovy/ApacheHttpAsyncClientCallbackTest.groovy

```diff
@@ -1,79 +0,0 @@
-/*
```

Comment:
Callback and null callback are both handled in the main test now, so deleted these but check my work that they were indeed redundant (AFAICT, one of the already was even before my PR)